site/app/libraries/Access.php

```diff
@@ -242,6 +245,18 @@ private function loadDirectories() {
                 "path.write" => "path.write.submissions",
             ]
         ];
+
+        $this->directories["banner_images"] = [
+            "base" => $this->core->getConfig()->getSubmittyPath() . "/banner_images",
+            "subparts" => [],
+            "permissions" => [
+                "path.read" => "path.read.submissions",
+                "path.write" => "path.write.submissions",
```

Comment:
Do we want students to write as well? Because, _$this->permissions["path.write.submissions"] = self::ALLOW_MIN_STUDENT | self::ALLOW_ONLY_SELF_GRADEABLE | self::CHECK_CSRF;_  allows Minimum Student